test/middleware/jsonld.test.ts

```diff
@@ -0,0 +1,107 @@
+import { parse as parseContentType } from 'content-type';
+import { Context as JsonLdContext } from 'jsonld/jsonld-spec';
+import { Context, Response } from 'koa';
+import jsonld from '../../src/middleware/jsonld';
+import createContext from '../context';
+import runMiddleware, { Next } from '../middleware';
+
+const makeRequest = async (next?: Next, jsonLdContext: JsonLdContext = {}): Promise<Response> => {
+  const context = createContext();
+
+  return runMiddleware(jsonld(jsonLdContext), context, next);
+};
+
+const next = (body: unknown, type: string) => async ({ response }: Context): Promise<void> => {
+  response.body = body;
+  response.type = type;
```

Comment:
weird that it is the `next` middleware setting the body, although it checks that the `jsonld` middleware waits on it. If `response` was correctly populated at the start of the test this dummy middleware would be a simpler empty function?

Reply:
Still need one case to make sure it runs after `next`.

Reply:
Yes, but if that part breaks you get 1 failing test instead of all of them.

Reply:
Understand; think changing it doesn't make it simpler though. Happy if you'd like to refactor.

Reply:
Not much value in changing towards either way for me, raised just for the sake of discussion.